doc/quickstart.rst

```diff
@@ -0,0 +1,379 @@
+their domains. The parametertypes are floats, integers and categorical parameters.
+
+We optimize a SVM with the parameters *kernel*, *C*, *gamma*, *coef0*, *degree* and *shrinking*, which
+are further explained in the documentation of sklearn. Note that modifying *C* can
+quickly lead to overfitting, which is why we use cross-validation to evaluate
+the configuration.
+
+Let's start by creating a ConfigSpace-object and adding the first hyperparameter: the choice of
+the kernel.
+
+.. literalinclude:: ../examples/svm.py
+   :lines: 60-65
+   :lineno-match:
+
+We can add Integers, Floats or Categoricals to the ConfigSpace-object all at
+once, by passing them in a list. 
+
+.. literalinclude:: ../examples/svm.py
+   :lines: 67-70
+   :lineno-match:
+
+Not every kernel uses all the parameters. The sklearn-implementation of the SVM accepts all parameters we are optimizing, but ignores all those incompatible with the chosen kernel.
+We can reflect this in optimization using **conditions** to deactivate parameters that are irrelevant to the current kernel.
+Deactivated parameters are not considered during optimization, limiting the search-space to reasonable configurations.
+This way human knowledge about the problem is introduced.
+
+.. literalinclude:: ../examples/svm.py
+   :lines: 72-78
+   :lineno-match:
+
+Conditions can be used for various reasons. The `gamma`-hyperparameter for
+example can be set to "auto" or to a fixed float-value. We introduce a parameter
+that is only activated if `gamma` is not set to "auto".
+
+.. literalinclude:: ../examples/svm.py
+   :lines: 80-88
+   :lineno-match:
+
+Of course we also define a function to evaluate the configured SVM on the IRIS-dataset.
+Some options, such as the *kernel* or *C*, can be passed directly.
+Others, such as *gamma*, need to be translated before the call to the SVM.
+
+.. literalinclude:: ../examples/svm.py
+   :pyobject: svm_from_cfg
+   :lineno-match:
+
+We register the function to a Target Algorithm Evaluator (TAE_), which communicates
+between the function to be optimized and *SMAC* by calling the function with the
+desired configuration and interpreting the output (in this case, simply the
+score). We also call the function to get the default-value.
+
+.. literalinclude:: ../examples/svm.py
+   :lines: 98-104
+   :lineno-match:
+
+We need a Scenario-object to configure the optimization-process.
+We provide a `list of possible options`__ in the scenario.
+
+__ scenario_
+
+The initialization of a scenario in the code uses the same keywords as a
+scenario-file, which we used in the branin example.
+
+.. literalinclude:: ../examples/svm.py
+   :lines: 91-96
+   :lineno-match:
+
+Now we're ready to create a *SMAC*-instance, which handles the Bayesian
+Optimization-loop and calculates the incumbent. 
+To automatically handle the exploration of the search space 
+and evaluation of the function, SMAC needs as inputs the scenario object 
+as well as the function evaluator.
```

Comment:
As far as I remember, it is possible to pass the function directly into SMAC, without creating a target function evaluator. For a quickstart I would prefer using this as it simplifies the learning curve.